v2/cpu.go

```diff
@@ -25,9 +25,12 @@ type CPU struct {
 
 func (r *CPU) Values() (o []Value) {
 	if r.Weight != nil {
+		// Converting cgroups configuration from v1 to v2
+		// more here https://github.com/containers/crun/blob/master/crun.1.md#cgroup-v2
+		convertedWeight := (1 + ((*r.Weight-2)*9999)/262142)
```

Comment:
I already did this in another PR, in the utils.go file there is a ToResources that has all the mappings from an OCI spec into the Resource type

Reply:
Oh, got it. Then we can close this one.